projects/core/src/occ/utils/index.ts

```diff
@@ -4,6 +4,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { InjectionToken } from '@angular/core';
+
+import * as OCC_USER_IDS from './occ-user-ids';
+
+export { OCC_USER_IDS };
+
 export * from './interceptor-util';
 export * from './occ-constants';
 export * from './occ-url-util';
+export * from './occ-asm-token';
+
+export const OCC_USER_ID_CONSTANTS = new InjectionToken<{
+  [identifier: string]: string;
+}>('List of OCC constants that pass for user IDs.');
```

Comment:
Couldn't we let other classes import the constants instead of requiring dependency injection for them to use the constants?